topics/proteomics/tutorials/metaquantome-data-creation/workflows/main_workflow.ga

```diff
@@ -40,18 +40,18 @@
             "inputs": [
                 {
                     "description": "",
-                    "name": "FASTA_db"
+                    "name": "ProteinDB_cRAP.fasta"
                 }
             ],
-            "label": "FASTA_db",
-            "name": "Input dataset",
+            "label": "ProteinDB_cRAP.fasta",
+            "name": "ProteinDB_cRAP.fasta",
             "outputs": [],
             "position": {
                 "left": 244.40625,
                 "top": 921.640625
             },
             "tool_id": null,
-            "tool_state": "{\"optional\": false}",
+            "tool_state": "{\"name\": \"ProteinDB_cRAP.fasta\"}",
```

Comment:
Do you really need to change the tool state.

Reply:
I'm going by the example [here](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-technical/tutorial.html#testing-the-workflow-recommended) which indicates to align the inputs name, label, name, and tool_state name:

```
{
    "id": 0,
    "input_connections": {},
    "inputs": [
        {
            "description": "",
            "name": "good_outcome"
        }
    ],
    "label": "good_outcome",
    "name": "good_outcome",
    "outputs": [],
    "position": {
        "left": 10,
        "top": 10
    },
    "tool_id": null,
    "tool_state": "{\"name\": \"good_outcome\"}",
    "tool_version": null
}
```